snyk-import/snyk-api.ts

```diff
@@ -0,0 +1,114 @@
+// deno-lint-ignore-file no-explicit-any
+
+import { API } from "./const.ts";
```

Comment:
Should maybe name this `SNYK_API_URI` or something that at least includes `SNYK`. We're dealing with both snyk and phylum here so it isn't always immediately obvious.